### PR TITLE
[CUBRIDMAN-166] serial.CURRVAL and serial.NEXTVAL are omitted

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CUBRIDMAN-166

### CODE REVIEW

File: ko/sql/schema/serial_stmt.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ CREATE SERIAL
 
 *   시리얼은 다중 사용자 환경에서 고유한 순번을 생성하는데 용이하다.
 *   시리얼 번호는 테이블과 독립적으로 생성된다. 따라서 하나 이상의 테이블에 동일한 시리얼을 사용할 수 있다.
-*   **PUBLIC** 을 포함하여 모든 사용자가 시리얼 객체를 생성할 수 있다. 일단 생성되면 모든 사용자들이 **CURRENT_VALUE**, **NEXT_VALUE** 를 통해 시리얼 숫자를 가져갈 수 있다.
+*   **PUBLIC** 을 포함하여 모든 사용자가 시리얼 객체를 생성할 수 있다. 일단 생성되면 모든 사용자들이 **CURRENT_VALUE (CURRVAL과 동일)**, **NEXT_VALUE (NEXTVAL 동일)** 를 통해 시리얼 숫자를 가져갈 수 있다.
```
Comment (mhoh3963) [inline ko/sql/schema/serial_stmt.rst]:
```suggestion
*   **PUBLIC** 을 포함하여 모든 사용자가 시리얼 객체를 생성할 수 있다. 일단 생성되면 모든 사용자들이 **CURRENT_VALUE (CURRVAL과 동일)**, **NEXT_VALUE (NEXTVAL과 동일)** 를 통해 시리얼 숫자를 가져갈 수 있다.
```

**Review by mhoh3963 — APPROVED** [review]

**Review by ssihil — APPROVED** [review]

File: en/sql/schema/serial_stmt.rst [hunk 2]
```diff
@@ -248,6 +251,9 @@ The following example shows how to create a table *athlete_idx* where athlete nu
             10004  'Choo'
             10006  'Lee'
 
+            serial_current_value(order_no)
+    ======================
+            10006
```
Comment (hgryoo) [inline en/sql/schema/serial_stmt.rst:256]:
```suggestion
            10006

```

File: ko/sql/schema/serial_stmt.rst [hunk 3]
```diff
@@ -249,6 +252,9 @@ DROP SERIAL
             10004  'Choo'
             10006  'Lee'
 
+            serial_current_value(order_no)
+    ======================
+            10006
```
Comment (hgryoo) [inline ko/sql/schema/serial_stmt.rst:257]:
```suggestion
            10006

```

**Review by hgryoo — COMMENTED** [review]
To fix warning: Explicit markup ends without a blank line; unexpected unindent.

**Review by sungc-cubrid — APPROVED** [review]

**Pull request merged**